packages/repository/src/__tests__/unit/repositories/crud.repository.unit.ts

```diff
@@ -4,26 +4,25 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {expect} from '@loopback/testlab';
-
+import {Callback} from 'loopback-datasource-juggler';
 import {
+  AnyObject,
+  Class,
+  Count,
+  CrudConnector,
   CrudRepository,
   CrudRepositoryImpl,
   DataSource,
   Entity,
   EntityData,
-  Class,
   Filter,
-  Where,
   Options,
-  CrudConnector,
-  AnyObject,
-  Count,
+  Where,
 } from '../../..';
-
 /**
  * A mock up connector implementation
  */
-class CrudConnectorStub implements CrudConnector {
+export class CrudConnectorStub implements CrudConnector {
```

Comment:
Please move this stub class into some kind of a helper file, so that tests don't import from each other. I think these stub should be used by unit-level tests only, integration-level and acceptance tests should use real connectors.

Proposed file name: `packages/repository/src/__tests__/unit/crud-connector.stub.ts`